source/_integrations/tuya.markdown

```diff
@@ -28,6 +28,20 @@ The Tuya integration allows integrating all Powered by Tuya devices you have add
 
 {% include integrations/config_flow.md %}
 
+Consult the following table and enter your Tuya credentials.
+
+| Field | Description |
```

Comment:
Please don't use tables for this (use `configuration_basic` blocks instead).

Additionally, this is information that should be showing in the dialogs. It is not common to add this to the integration description page.

Reply:
Deleted